homeassistant/components/reolink/__init__.py

```diff
@@ -175,3 +178,56 @@ async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
         hass.data[DOMAIN].pop(config_entry.entry_id)
 
     return unload_ok
+
+
+def cleanup_disconnected_cams(
+    hass: HomeAssistant, config_entry_id: str, host: ReolinkHost
+) -> None:
+    """Clean-up disconnected camera channels or channels where a different model camera is connected."""
```

Comment:
Please break long strings at max 88 characters per line. According to PEP8, docstrings should not be longer than 72 characters per line. The header should be a single sentence on a single line and then an optional body, after a blank line, is possible.

Reply:
Fixed in this PR: https://github.com/home-assistant/core/pull/104793